src/schema/objects/entities.yaml

```diff
@@ -422,3 +422,22 @@ tracksys:
     may be longer and more human readable.
   type: string
   format: label
+stimulus:
+  name: stim
+  display_name: Stimulus
+  description: |
+    The `stim-<label>` entity can be used to distinguish stimulus files stored in the `stimuli/` directory.
+
+    This entity is only applicable to stimulus files.
+  type: string
+  format: label
+annotation:
+  name: annot
+  display_name: Annotation
+  description: |
+    The `annot-<label>` entity can be used to distinguish different annotations
+    of the stimulus data.
+
+    This entity is only applicable to stimulus files.
```

Comment:
A parallel to investigate here is `_events.tsv` files, i.e. those could also be created/annotated across various dimensions. ATM they would just "flatten" them into this single file with some column potentially disentangling those dimensions for different annotations (e.g. low level properties of visual stimuli vs higher level cognitive concepts vs responses).  So it feels to me it might be applicable there too.

But with that it somewhat begs me to think about some possible alternative name/concept but I find none and there be none ;)

Reply:
Since we already ran into discussion either `desc` could be in effect used as `desc` (and we have example in bids of `_desc-autoannot`), it would be valuable to describe here the difference from `desc`, e.g. `annot` is more specialized in providing annotations to the stimuli/events whenever `desc` is more about processing of the data.  So `_annot-chatgpt` would be preferable over `_desc-chatgpt` as would symbolize annotation over some other chatgpt driven processing (e.g. prompted chatgpt for some analysis of data/stimuli)